src/main/java/locustTask/ForcedNo.java

```diff
@@ -0,0 +1,51 @@
+package locustTask;
```

Comment:
The package should be named in lower case as convention:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html